src/python/visclaw/colormaps.py

```diff
@@ -29,35 +29,41 @@ def make_colormap(color_list):
     html hex string, e.g. "#ff0000" for red.
     """
 
```

Comment:
Can you add documentation to this function pointing out that alpha channels have been added as an optional specification?